src/base/memory_monitor_common.h

```diff
@@ -26,13 +26,11 @@
 
 #include <cstdint>
 
-typedef struct memmon_mem_stat
+typedef enum
 {
-  uint64_t init_stat;
-  uint64_t cur_stat;
-  uint64_t peak_stat;
-  uint32_t expand_count;
-} MEMMON_MEM_STAT;
+  HEAP_MODULE = 1,
+  MMM_MODULE_END
```

Comment:
```suggestion
  MMM_MODULE_LAST
```

Reply:
How about using all or none of the MMM_ prefixes for code consistency? I mean MMM_HEAP_MODULE or MODULE_LAST.

Reply:
I modified HEAP_MODULE->MMM_HEAP_MODULE and MMM_MODULE_END->MMM_MODULE_LAST.